### ISSUE TITLE
[a11y] Cannot change user role name

### ISSUE THREAD

**Issue by grantfitzsimmons** [specify/specify7#1509]
![msedge_Gh9VryrJtd](https://user-images.githubusercontent.com/37256050/167709975-ca3d88df-d93f-46d9-b999-6672d1f06a56.gif)

Trying to change the user role name results in a **Page Not Found** error.
